src/lzf/lzfP.h

```diff
@@ -144,7 +144,9 @@ using namespace std;
 # if defined (WIN32)
 #  define LZF_USE_OFFSETS defined(_M_X64)
 # else
-#  if __cplusplus > 199711L
+#  if MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_8
```

Comment:
It definitely should, I'm just trying to help the community transition to
Node 4.0. I will try to do this later today.

On Mon, Sep 21, 2015 at 9:15 AM, Ian Babrou <notifications@github.com>
wrote:

> In src/lzf/lzfP.h
> <https://github.com/Topface/node-lzf/pull/9#discussion_r39968314>:
>
> > @@ -144,7 +144,9 @@ using namespace std;
> >  # if defined (WIN32)
> >  #  define LZF_USE_OFFSETS defined(_M_X64)
> >  # else
> > -#  if __cplusplus > 199711L
> > +#  if MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_8
>
> It still changes liblzf and I still believe that this change should go
> upstream (liblzf itself)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Topface/node-lzf/pull/9/files#r39968314>.
>